### DISCUSSION TITLE
config.ini? (BTT TFT24)

### DISCUSSION THREAD

**Discussion by ubiratamuniz** [teeminus/NoTouchScreenFirmware#75] [Q&A]
Howdy, 

I´ve downloaded the binary file for the TFT24 screen from Bigtreetech. I put the BIGTREE_TFT24_V1.1.26.x.bin file (no changes in the name or case of the letters) on the SD card (a 2GB SD card formatted in FAT32) and nothing happens when I boot up the screen or press the reset button. It does recognize the card inserted but does not try to update the firmware. 

If I try to use the BigTreeTech config.ini file (from their github) along with the binary (I tried it in order to test it out), it tries to update the firmware, but it gives a lot of keyword errors (as expected as it´s a different firmware) and aborts the update. However, I didn´t add the additional resources such as images and such as I believe they won´t be used in the NoTouch firmware. Should I add them as well? 

So, it seems that the screen requires a config.ini file in the root of the SD card along with the binary in order for the firmware to start the update.

Which would be the proper config.ini file for NoTouchScreenFirmware? What else can I try?

**Comment by teeminus** [comment]
Hi, there is no config file need for updating the firmware.

This firmware is quite outdated. Chances are, that there is a new hardware revion or some firmware changes in the original BTT firmware which require a different firmware file name.

**Reply by ubiratamuniz** [reply]
I see. Current version on BTT github is BIGTREE_TFT24_V1.1.27.x.bin (which unless I'm mistaken is the firmware version installed in my display). So, considering the naming format is correct, is it possible that the board simply doesn't accept a backwards version?

**Reply by teeminus** [reply]
Yeah, this could be true. You could try renaming the firmware update file and check if that works. On the other hand, there might be an other reason why they changed the firmware update file name. Maybe due to hardware changes. In other words, there is a chance that updating the display with a renamed update file might brick the display.
